_i18n/en.yml

```diff
@@ -326,13 +336,23 @@ help_pages:
 
       [Why do I need to use login.gov to access government services online?](site.baseurl/help/creating-an-account/why-do-i-need-to-use-logingov-to-access-government-services-online/)
     what-is-my-personal-key: |-
-      A personal key is a 16-character secret code we give you when you create an account. The personal key is the only way to sign in if you lose access to the device you use for two-factor authentication.
+      A personal key is a 16-character secret code we have been historically generating as a form of two factor authentication. [Read more about two-factor authentication here](site.baseurl/help/creating-an-account/what-is-two-factor-authentication/)
 
-      You can only use your personal key once.  You will receive a new personal key each time you use it to sign in, and your old one will no longer work.
+      Recently we have began the process of retiring personal keys as a form of two factor authentication as they are apt to being lost easily. In their place users now have a variety of options for a second authentication factor including backup codes.
 
-      Because a personal key is the only way to recover your account, make sure to store it in a safe place. Some good ways of storing your personal key include writing it down, taking a screenshot, pasting it into a secure document, or making a printout of it.
+      If you are creating a new account you will not be assigned a personal key. Instead, select one of the available two-factor authentication options presented.
+
+      If you were assigned a personal key previously you are still able to use it, however next time you use the personal key you will be asked to select a new form of two factor authentication and the personal key option will be removed. Until you use your current personal key it will still be active and available to you as a two factor option.
+
+      <!---
+      TO ADD WHEN DEPLOYING DOCAUTH
+      You are assigned a personal key when you complete your identification proofing. You can only use your personal key once.  You will receive a new personal key each time you use it to sign in, and your old one will no longer work.
+
+      Because a personal key is the only way to reset your password while maintaining your identity validated, make sure to store it in a safe place. Some good ways of storing your personal key include writing it down, taking a screenshot, pasting it into a secure document, or making a printout of it. If you do forget your personal key you will still be able to recover your account but must go through the proofing process again.
```

Comment:
put a comma between "if you do forget your personal key" and "you"

Reply:
Fixed